.github/workflows/build.yaml

```diff
@@ -11,7 +11,7 @@ jobs:
     strategy:
       matrix:
         bot: ['base']
-        lang: ['eng', 'afr']
+        lang: ['eng', 'afr', 'xho']
```

Comment:
We probably don't need this on the build step as all the models are the same. We can just reference the english model in all the workers, thoughts?

Reply:
No, the domain file is included in the model, so we have to build a model for each language.